### DISCUSSION TITLE
Incoming issue Neither showing on UI nor getting Bounce,

### DISCUSSION THREAD

**Discussion by rccess** [postalserver/postal#1531] [Help with using Postal]
Hello Community 

I have Setup one Domain 

![image](https://user-images.githubusercontent.com/32220165/128383881-5ebb950f-5876-4fe6-aa9f-ff2e96e630ed.png)

Return path, MX and other records properly setup

![image](https://user-images.githubusercontent.com/32220165/128384059-3c602153-011c-45c3-9f71-c1eb417f370d.png)

Setup email Routing for domain info@care.mailxxxxxx.hoxx to my gmail id : helxxxx@gmail.com

![image](https://user-images.githubusercontent.com/32220165/128384147-b820cd14-f92a-438e-b9f3-3a7f9fc8e0c6.png)

![image](https://user-images.githubusercontent.com/32220165/128384237-ace79936-25f2-4d13-9a34-66b9ff2065cf.png)

MX Records Properly Setup 
![image](https://user-images.githubusercontent.com/32220165/128384332-fb3dd075-bfa4-488f-b851-cf27b71f5360.png)

Here is the config file 

![image](https://user-images.githubusercontent.com/32220165/128384557-a7561021-40cf-449d-9848-0d681842a673.png)

When i am sending email from my gmail to my setup domain id : info@care.mailxxxxxx.hoxx no  email showing in UI and as well not receiving on gmail which setup as routing email

Please do advise

**Comment by willpower232** [comment]
it looks like you are missing the A record pointing mx.rmta.whatever.care to your Postal server.

**Reply by rccess** [reply]
working fine, thank you :)

**Reply by rccess** [reply]
@willpower232 

i have noticed that 
all emails forwarding to routing emails from main ip of the server
how we can send send with the IP which organization using by default

**Reply by willpower232** [reply]
it sounds like you want to take a look at IP Pools https://docs.postalserver.io/features/ip-pools

**Reply by fiantinangon** [reply]
> it looks like you are missing the A record pointing mx.rmta.whatever.care to your Postal server.

I've exactly same issue, so I need to add A record and pointing mx.postal.example.com to same IP with postal.example.com  ? @willpower232 

@rccess would you please share what's your missing on your DNS configuration ?

<img width="622" alt="Screen Shot 2021-10-26 at 08 45 01" src="https://user-images.githubusercontent.com/9161278/138794898-30e470a0-72cd-4cc9-9247-d2ab4583c641.png">
<img width="1231" alt="Screen Shot 2021-10-26 at 08 44 49" src="https://user-images.githubusercontent.com/9161278/138794906-a3647fe5-57a3-49ef-99cf-4626f70973ae.png">
